dashboard-templates/frontend.gitlab-ci.yml

```diff
@@ -16,7 +16,7 @@ default:
 build_package:
   stage: build
   tags:
-    - saas-linux-medium-amd64
+    - amd64
```

Comment:
Keep the runner tag for build_package job as `saas-linux-medium-amd64`. Our custom runner doesn't have enough RAM to build node packages.